### PR TITLE
Add CODEFLARE.md

### PR DESCRIPTION
**Pull request by tardieu**
Add documentation to help transition from CodeFlare stack to MLBatch.

### CODE REVIEW

File: CODEFLARE.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+Borrowed GPU count:                 8
+```
+The tool enumerates the cluster queues defined on the cluster showing the GPU
+quota for each team as well as the number of GPUs in use by admitted workloads.
+The GPU usage may exceed the GPU quota for a team if this team is borrowing idle
+capacity from the cluster.
+
+The tool also reports the total GPU capacity distinguishing healthy (i.e.,
+schedulable, available for use) and unhealthy (i.e., unschedulable, unavailable)
+GPUs. The nominal GPU quota represents to cumulative GPU quota across all the
```
Comment (dgrove-oss) [inline CODEFLARE.md:70]:
```suggestion
GPUs. The nominal GPU quota represents the cumulative GPU quota across all the
```

File: CODEFLARE.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+kubectl apply -f wrapped-pod.yaml
+```
+
+MLBatch includes an [appwrapper-packager](tools/appwrapper-packager/) tool to
+automate the addition this prefix as well as the indentation of the workload
+specification. In addition, MLBatch includes a new implementation of the
+[pytorch-generator](tools/pytorchjob-generator/) tool to facilitate the
+configuration of PyTorch jobs including the addition of the AppWrapper prefix.
+
+As a result of the AppWrapper simplification for MLBatch, AppWrappers now in
```
Comment (dgrove-oss) [inline CODEFLARE.md:147]:
```suggestion
As a result of the AppWrapper simplification for MLBatch, AppWrappers which are now in
```

**Review by dgrove-oss — APPROVED** [review]
two suggested small edits.  otherwise lgtm

File: CODEFLARE.md (hunk 1 above)
Comment (tardieu) [reply CODEFLARE.md]: fixed

File: CODEFLARE.md (hunk 2 above)
Comment (tardieu) [reply CODEFLARE.md]: fixed

**Pull request merged**